ajax/libs/matterjs/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "matterjs",
+  "filename": "matter.js",
+  "version": "0.1.3",
+  "description": "A tiny collection of UI components",
+  "keywords": [
+    "matter",
+    "matterjs",
+    "components",
+    "deku",
+    "deku-component",
+    "ui"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/stevenmiller888/matter.git"
+  },
+  "autoupdate": {
```

Comment:
@anubhabsen The auto-updater will not work because there is no git tag on its git repo. Let's transfer to use npm auto-update config instead. Thanks.